cypress/pages/settings-public-api.ts

```diff
@@ -0,0 +1,11 @@
+import { BasePage } from './base';
+
+export class PublicApiPage extends BasePage {
+	url = '/settings/api';
+	secret = '';
+
+	getters = {
+		upgradeCTA: () => cy.getByTestId('public-api-upgrade-cta'),
+	};
+	actions = {};
+}
```

Comment:
Sorry for the nitpick, but can you please add a function instead? We'll be migrating slowly to functions instead of page objects. https://www.notion.so/n8n/E2E-Testing-2423349afab34bb5b8a7241e7b4cb54f?pvs=4#1af52ce8de40446c802461d2475d25ed

Reply:
Do we have a project in linear where we are tracking the "migration"?

Reply:
Not yet, it's all rather new. Thanks for taking care of it 🙏

Reply:
want me to create one?